apex/contrib/optimizers/distributed_fused_adam.py

```diff
@@ -86,6 +96,56 @@ class DistributedFusedAdam(torch.optim.Optimizer):
 
     """
 
+    class ParameterFragment():
```

Comment:
we don't need an empty `()` per https://docs.python.org/3/tutorial/classes.html#class-definition-syntax.

```suggestion
    class ParameterFragment:
```